corehq/apps/reports/templates/reports/tableau_visualization_detail.html

```diff
@@ -0,0 +1,10 @@
+{% extends "hqwebapp/base_section.html" %}
```

Comment:
Seems like this and `tableau_server.html` could be the same template. Could you extract a template called something like `crispy_form.html` or `single_crispy_form.html` and stick it in the `hqwebapp` django app?

There are about 20 other files that could probably take advantage of such a template (pasting here for future reference, not asking you to update them):
corehq/apps/accounting/templates/accounting/accounts_base.html
corehq/apps/accounting/templates/accounting/plans_base.html
corehq/apps/accounting/templates/accounting/subscriptions_base.html
corehq/apps/accounting/templates/accounting/test_reminder_emails.html
corehq/apps/accounting/templates/accounting/trigger_accounting_tests.html
corehq/apps/accounting/templates/accounting/trigger_bookkeeper.html
corehq/apps/accounting/templates/accounting/trigger_customer_invoice.html
corehq/apps/accounting/templates/accounting/trigger_invoice.html
corehq/apps/case_migrations/templates/case_migrations/migration.html
corehq/apps/domain/templates/domain/admin/calendar_fixture.html
corehq/apps/domain/templates/domain/admin/commtrack_settings.html
corehq/apps/domain/templates/domain/admin/location_fixture.html
corehq/apps/domain/templates/domain/admin/openclinica_settings.html
corehq/apps/domain/templates/domain/admin/project_privacy.html
corehq/apps/domain/templates/domain/admin/transfer_domain.html
corehq/apps/integration/templates/integration/dialer_settings.html
corehq/apps/integration/templates/integration/gaen_otp_server_settings.html
corehq/apps/integration/templates/integration/hmac_callout_settings.html
corehq/apps/locations/templates/locations/filter_and_download.html
corehq/apps/ota/templates/ota/prime_restore_cache.html
corehq/apps/sms/templates/sms/subscribe_sms.html
corehq/apps/sso/templates/sso/accounting_admin/new_identity_provider.html
corehq/motech/dhis2/templates/dhis2/dataset_map_create.html

Reply:
Created template in this commit: https://github.com/dimagi/commcare-hq/pull/29946/commits/4cde39f69f84ffe870464060e05a4e23ebe3260f